perf/common.jl

```diff
@@ -12,14 +12,9 @@ The configuration precedence is as follows:
 There is an exception for the job `flame_perf_target_edmf`, which requires that 
```

Comment:
The docstring should be updated

Reply:
I will clean this up, just wanted to run CI to ensure the config is being set correctly. Thanks for looking it over!